jest.config.electron.e2e.ts

```diff
@@ -0,0 +1,27 @@
+import { defaults } from "jest-config";
+
+const jestConfig = {
+	...defaults,
+	roots: ["<rootDir>/src/electron"],
+	testMatch: ["**/e2e/**/*.test.ts"],
```

Comment:
shoul this rather be `*.e2e.test.ts`? (see comment beolw)

Reply:
I decided against having `e2e` in the file name, but have a dedicated folder for e2e-tests.